environment/prod/deployment/prod/docker-compose.prod.yml

```diff
@@ -23,7 +23,7 @@ services:
         max-file: "5"
     deploy:
       mode: replicated
-      replicas: 1
+      replicas: 4
```

Comment:
What is the use case?
I think it's not a good idea. It will create 4 containers.

Reply:
only for testing and an error appeared there. I wanted to check this because deploy app 1 replaces app 2, which was previously deployed